### PR TITLE
[#1] Generate new project from Flutter template

### LINKED ISSUE #1: [Chore] Generate a new Flutter project from the template
## Why

To quickly kick off the IC project, we must use the Nimble Flutter Template (https://github.com/nimblehq/flutter_templates)

## Acceptance Criteria

- [ ] Generate a new project using the flutter template
  - Package name must be `com.lydiaryan.fluttersurvey`
  - App name must be `Flutter Survey - Lydia Ryan`

### PR DESCRIPTION
**Pull request by ryan-conway**
closes #1 

## What happened 👀

- Generated new project using [Flutter template](https://github.com/nimblehq/flutter_templates)

## Insight 📝

N/A

## Proof Of Work 📹

project runs successfully 

Screenshot_2023-03-08-16-42-48-09_5b336af4fc7e8977380a7e7783c0bdb5

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+https://github.com/nimblehq/flutter_templates/issues/??
```
Comment (lydiasama) [inline .github/PULL_REQUEST_TEMPLATE.md]: Let's add this:
```suggestion
- Closes https://github.com/nimblehq/flutter-ic-lydia-ryan/issues/??
```

File: .github/ISSUE_TEMPLATE/feature_template.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (lydiasama) [inline .github/ISSUE_TEMPLATE/feature_template.md:1]: Do we need this? I think we use only `story_template` to generate our feature task. 🤔

**Review by lydiasama — COMMENTED** [review]
Shall we adopt this `.github/workflows/draft-new-release.yml` from https://github.com/nimblehq/android-templates/pull/245/files to help draft the release?

**Comment by lydiasama** [comment]
The rest LGTM. 🚀

File: .github/ISSUE_TEMPLATE/feature_template.md (hunk 2 above)
Comment (ryan-conway) [reply .github/ISSUE_TEMPLATE/feature_template.md:1]: This is all from the template with nothing removed 😅 Shall we remove this in #6? 😄

Comment (lydiasama) [reply .github/ISSUE_TEMPLATE/feature_template.md:1]: @ryan-conway Sure thing. 😬

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (lydiasama) [reply .github/PULL_REQUEST_TEMPLATE.md]: @ryan-conway Let's update the repository link too. 😬

**Review by lydiasama — APPROVED** [review]

**Review by markgravity — APPROVED** [review]

**Pull request merged**